exploring_UV_extinction_curves/Exploring_UV_extinction_curves.ipynb

```diff
@@ -0,0 +1,1662 @@
+{
```

Comment:
<ul><li>Add another sentence about the physical scenarios where extinction is relevant. eg., sometimes the dust is due to dust in the galaxy that just happens to be between the observer and the star, sometimes it is dust very near to the star due to it's own disk or proto-stellar cloud.</li><li>Put equation last, after definitions.</li><li>Add more words about the naught subscript. E.g., in the definition of color index: (B-V) is the observed color index and (B-V)_0 is what the color would be if there no extinction due to dust.</li><li>cut FITS file definition.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>